auth-web/src/components/auth/staff/continuation-in/StaffContinuationInReview.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <v-container class="pa-0">
+    <header class="view-header align-center justify-space-between mt-n1 mb-4">
+      <h2 class="view-header__title">
+        Continuation Applications
+      </h2>
+    </header>
+
+    <!-- Table Contents -->
+    <StaffContinuationInTable />
+  </v-container>
+</template>
+
+<script lang="ts">
+
+import { Component, Vue } from 'vue-property-decorator'
+import StaffContinuationInTable from '@/components/auth/staff/continuation-in/StaffContinuationInTable.vue'
```

Comment:
In this case I think  you can get away with `from './StaffContinuationInTable.vue'`.

Also, we've been moving away from "continuation in" and towards "continuation application". If it's not too messy, I suggest changing the folder and component names accordingly.

Reply:
Updated